docs/antrea-network-policy.md

```diff
@@ -1066,14 +992,26 @@ spec:
       matchLabels:
         app: client
   egress:
-  - action: Drop
+  - action: Allow
     to:
       - fqdn: "*foobar.com"
+    ports:
+      - protocol: TCP
+        port: 8080
+  - action: Drop      # Drop all other egress traffic, in-cluster or out-of-cluster 
 ```
 
-The above example drops all traffic destined to any FQDN that matches the wildcard
-expression `*foobar.com` originating from any Pod with label `app` set to `client`
-across any Namespace. This feature only works at the L3/L4 level.
+The above example allows all traffic destined to any FQDN that matches the wildcard expression
+`*foobar.com` on port 8080, originating from any Pod with label `app` set to `client`
+across any Namespace. For these `client` Pods, all other egress traffic are dropped.
+
+Note that for FQDN wildcard expressions, the `*` character can match multiple subdomains (i.e.
+`*foobar.com` will match `foobar.com`, `www.foobar.com` and `test.uswest.foobar.com`).
+
+This feature only works at the L3/L4 level, which means that Antrea will only program datapath
```

Comment:
I think we can remove "works at the L3/L4 level". Just say it will not apply to DNS packets.

Reply:
Done